packages/schema-generator/API.md

```diff
@@ -0,0 +1,47 @@
+## API Report File for "@aws-amplify/schema-generator"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+// @public (undocumented)
+export type AmplifyGenerateSchemaError = 'DatabaseConnectionError' | 'DatabaseUrlParseError';
+
+// @public
+export abstract class DatabaseConnectError extends Error {
+    constructor(code?: string | undefined);
+    // (undocumented)
+    readonly code?: string | undefined;
+}

```

Comment:
if we're not using this type outside of new package we should hide it from public api.